plugins/ltac/tactic_debug.ml

```diff
@@ -370,7 +370,10 @@ let explain_ltac_call_trace last trace loc =
           strbrk " (with " ++
             prlist_with_sep pr_comma
             (fun (id,c) ->
-              let sigma, env = Pfedit.get_current_context () in
+              (* XXX: This hooks into the ExplainErr extension API
+                 so it is tricky to provide the right env for now. *)
+              let env = Global.env () in
```

Comment:
@ppedrot is this change ok with you? It does not seem easy to do better at the moment, but I'm not 100% how serious this is.

Reply:
Tactic debugging is already half broken, so I wouldn't expect much from this. At worst you get to print closure variables of type `constr` in a wrong env, but the new one doesn't seem wronger than the previous one (think about closures capturing variables in a different proof state).